### DISCUSSION TITLE
我相信未來一切會更好──約哈娜與媽媽的小豬農場

### DISCUSSION THREAD

**Discussion by Guagua1234** [1101-GS4719-C/Python-Community-GS4719-C#1138] [General]
https://www.thenewslens.com/feature/2021-worldvision/169933
我認為約哈娜和她的家人住在一个破旧的小木屋中，他们努力工作希望能够改善生活状况。我覺得媽媽會希望女兒有成功的人生，并且会尽可能地支持她們。而約哈娜希望将来能够有很多又大又多的猪，借助媽媽的专业知识开一家专门卖猪肉的商店并成为村庄的先驱。

**Comment by 110302550** [comment]
我覺得約哈娜和嬤嬤兩個人相知相惜的情感令我很感動，媽希望約哈娜可以完成夢想，約哈娜則希望媽媽可以養很多豬，系望他們的未來真的能如他們所說的都變好
